src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml

```diff
@@ -52,6 +52,7 @@ tests:
           value: 10
           constraints:
               type: uint16
+              notValue: OnOffTransitionTimeValue
```

Comment:
Why not?  What if the original value was 10?

Reply:
@bzbarsky-apple i will update the script with configurable values

Reply:
@bzbarsky-apple[ TC-LVL-2.2](https://github.com/project-chip/connectedhomeip/blob/53415c2ad4cfa98f51128209b5060427ac2dbc71/src/app/tests/suites/certification/Test_TC_LVL_2_2.yaml) is modified and rebased can you please re-verify it